tests/model_serving/model_server/conftest.py

```diff
@@ -371,18 +374,23 @@ def ovms_raw_inference_service(
     ovms_kserve_serving_runtime: ServingRuntime,
     ci_endpoint_s3_secret: Secret,
 ) -> Generator[InferenceService, Any, Any]:
-    with create_isvc(
-        client=unprivileged_client,
-        name=f"{request.param['name']}-raw",
-        namespace=unprivileged_model_namespace.name,
-        external_route=True,
-        runtime=ovms_kserve_serving_runtime.name,
-        storage_path=request.param["model-dir"],
-        storage_key=ci_endpoint_s3_secret.name,
-        model_format=ModelAndFormat.OPENVINO_IR,
-        deployment_mode=KServeDeploymentType.RAW_DEPLOYMENT,
-        model_version=request.param["model-version"],
-    ) as isvc:
+    isvc_kwargs = {
+        "client": unprivileged_client,
+        "name": f"{request.param['name']}-raw",
+        "namespace": unprivileged_model_namespace.name,
+        "external_route": True,
+        "runtime": ovms_kserve_serving_runtime.name,
+        "storage_path": request.param["model-dir"],
+        "storage_key": ci_endpoint_s3_secret.name,
+        "model_format": ModelAndFormat.OPENVINO_IR,
+        "deployment_mode": KServeDeploymentType.RAW_DEPLOYMENT,
+        "model_version": request.param["model-version"],
+    }
+
+    if (stop_resume := request.param.get("stop")) is not None:
```

Comment:
Instead of changing it, you may do it similarly to my previous suggestion:
```
with create_isvc(
  ...
  stop_resume=request.param.get("stop", False)
)
```